### PR TITLE
Consume dataviz into cards

### PR DESCRIPTION
**Pull request by nulikartik**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes
Adding donut, pie, stacked bar chart variants and multicount to the card. Stacked bar chart has two variations. These variations appear depending upon the number of data points passed to it.
image
image
image
image

### CODE REVIEW

File: packages/dashboard/src/components/Card/CardPage.tsx [hunk 1]
```diff
@@ -4,10 +4,18 @@ import { SmallCardBasicExample } from './examples/Card.Small.Basic.Example';
 import { MediumTallCardBasicExample } from './examples/Card.MediumTall.Basic.Example';
 import { MediumWideCardBasicExample } from './examples/Card.MediumWide.Basic.Example';
 import { LargeCardBasicExample } from './examples/Card.Large.Basic.Example';
+import { DonutAndPieChartExample } from './examples/DonutAndPieChartExample';
+import { LineAndVerticalBarChartExample } from './examples/LineAndVerticalBarChartExample';
+import { MultiCountExample } from './examples/MultiCountExample';
+import { StackedBarChartExample } from './examples/StackedBarChartExample';
```
Comment (samiyaakhtar) [inline packages/dashboard/src/components/Card/CardPage.tsx]: nit: Can we rename these files to follow the standard Card.Chart.Donut.Example, Card.Chart.Bar.Example, Card.Small.MultiCount.Example, Card.Chart.StackedBar.Example

File: packages/dashboard/src/components/Card/examples/LineAndVerticalBarChartExample.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+import { ICardProps, CardSize, Priority, CardContentType } from '../Card.types';
+import { Card } from '../Card';
```
Comment (samiyaakhtar) [inline packages/dashboard/src/components/Card/examples/LineAndVerticalBarChartExample.tsx]: Can you fix the imports to use something like 
`
import {
  Card,
  CardContentType,
  CardSize,
  ChartType,
  DashboardGridBreakpointLayouts,
  DashboardGridLayout,
  IAction,
  IThumbnailItemProps,
  Priority,
  Size
} from '@uifabric/dashboard';
`

File: packages/dashboard/src/components/Card/examples/MultiCountExample.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import { ICardProps, CardSize, Priority, CardContentType } from '../Card.types';
+import { Card } from '../Card';
```
Comment (samiyaakhtar) [inline packages/dashboard/src/components/Card/examples/MultiCountExample.tsx]: Same, fixing imports here will help others use the examples better

File: packages/dashboard/src/components/Card/examples/MultiCountExample.tsx [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+      {
+        title: 'Action 8',
+        action: () => {
+          alert('Action8 clicked');
+        }
+      },
+      {
+        title: 'Action 9',
+        action: () => {
+          alert('Action9 clicked');
```
Comment (samiyaakhtar) [inline packages/dashboard/src/components/Card/examples/MultiCountExample.tsx]: nit: we have 9 action bar buttons in each card, leave all 9 of them in one remove the unnecessary 7 from others? Trying to clean up examples to show what they're intended to show

Comment (nulikartik) [reply packages/dashboard/src/components/Card/examples/MultiCountExample.tsx]: removed action bar

File: packages/dashboard/src/components/Card/examples/LineAndVerticalBarChartExample.tsx (hunk 2 above)
Comment (nulikartik) [reply packages/dashboard/src/components/Card/examples/LineAndVerticalBarChartExample.tsx]: made import changes to previous examples as well.

File: packages/dashboard/src/components/Card/CardPage.tsx (hunk 1 above)
Comment (nulikartik) [reply packages/dashboard/src/components/Card/CardPage.tsx]: done

File: packages/dashboard/src/components/Card/examples/MultiCountExample.tsx (hunk 3 above)
Comment (nulikartik) [reply packages/dashboard/src/components/Card/examples/MultiCountExample.tsx]: done

**Pull request merged**